perun-core/src/test/java/cz/metacentrum/perun/core/entry/UsersManagerEntryIntegrationTest.java

```diff
@@ -701,6 +706,131 @@ public void isAlreadyReservedLogin() throws Exception {
 		 }
 		 */
 
+	@Test
+	public void getAllBlockedLoginsInNamespaces() throws Exception {
+		System.out.println(CLASS_NAME + "getAllBlockedLoginsInNamespaces");
+
+		perun.getUsersManager().blockLogin(sess, "login", null);
+		perun.getUsersManager().blockLogin(sess, "login", "namespace");
+		perun.getUsersManager().blockLogin(sess, "login2", "namespace2");
+
+		List<Pair<String, String>> listOfBlockedLogins = perun.getUsersManager().getAllBlockedLoginsInNamespaces(sess);
+		assertEquals(listOfBlockedLogins.size(), 3);
```

Comment:
I think we might want to check that obtained blocked login-namespace pairs really are those above with correct login value and namespace.